src/cli/bruker2dicom

```diff
@@ -58,13 +61,34 @@ def main():
     logging.getLogger().setLevel(verbosity.upper())
     
     function = arguments.pop("function")
+    source = arguments.pop("source")
+    
+    #create tmp directory in the current location
+    extractdir = tempfile.mkdtemp("","",".")
```

Comment:
Ok but letting it like that will create the directory under /tmp so it can cause problems for Windows users?